docker/php-fpm/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM phpdockerio/php56-fpm:latest
+WORKDIR "/application"
+
+# Fix debconf warnings upon build
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install selected extensions and other stuff
+RUN apt-get update \
+    && apt-get -y --no-install-recommends install  php5-mysql php5-redis php5-sqlite php5-gd \
+    && apt-get clean; rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
+
+# Install git
+RUN apt-get update \
+    && apt-get -y install git \
+    && apt-get clean; rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
```

Comment:
The Dockerfile should probably execute `composer install`, especially to avoid issues with the host using a different version of PHP.

Reply:
fixed